src/App.js

```diff
@@ -0,0 +1,84 @@
+import React, {useReducer, useState} from 'react';
+import logo from './logo.svg';
+import './App.css';
+import ToDos from './components/ToDos';
+import ToDoForm from './components/ToDoForm';
+import Footer from './components/Footer';
+import arrow from './arrow.png';
+import moment from 'moment';
+
+import { reducer, initialState } from './reducers/reducer';
+
+
+function App() {
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const today = moment();
+  const [todoInput, setTodoInput] = useState('');
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    console.log('in handleSubmit');
+    dispatch({type: 'ADD', payload: e.target.todo.value});
+    setTodoInput('');
+  }
+
+  const handleChanges = e => {
+    setTodoInput(e.target.value);
+  }
+
+  const toggleCompleted = (event, id) => {
+    event.target.classList.toggle('completed');
+    dispatch({type: 'TOGGLE', payload: id});
+    const todoEl = document.getElementById(id);
```

Comment:
Really nice! If you were to test this I might split up the reducer into two separate ones so they're a bit easier to test.